src/main/java/com/sprint/mission/discodeit/dto/BinaryContentDTO.java

```diff
@@ -0,0 +1,19 @@
+package com.sprint.mission.discodeit.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.UUID;
+
+public class BinaryContentDTO {
+
+    @Data
+    @AllArgsConstructor
+    @NoArgsConstructor
+    public static class CreateBinaryContentDTO {
+        private UUID userId;  // 해당 바이너리를 업로드한 사용자
+        private byte[] content;  // 바이너리 데이터
+        private String contentType;  // 파일 타입 (예: 이미지, PDF 등)
+    }
+}
```

Comment:
굳이 중첩클래스로 만들지 않아도 될 것 같아요.  

```suggestion
@Getter
@AllArgsConstructor
public class BinaryContentDTO {
    private UUID userId;
    private byte[] content;
    private String contentType;
}
```